code/game/machinery/vending/vendor_types/squad_prep/squad_engineer.dm

```diff
@@ -206,5 +206,5 @@ GLOBAL_LIST_INIT(cm_vending_clothing_engi, list(
 		/obj/item/stack/sheet/plasteel/medium_stack,
 		/obj/item/cell/high,
 		/obj/item/tool/shovel/etool/folded,
-		/obj/item/device/lightreplacer,
+		/obj/item/weapon/gun/smg/nailgun/compact/tactical,
```

Comment:
Either add this in addition to the existing light replacer entry here, or swap it with the entry in  `/obj/structure/machinery/cm_vending/sorted/tech/comtech_tools/populate_product_list`

Every comtech should be vended the light replacer; especially if I am going to further push the usage of colony lights. Up to them if they want to leave it.